test/src/hls-adapter.spec.js

```diff
@@ -210,29 +210,6 @@ describe('HlsAdapter Instance - Unit', function () {
     JSON.parse(JSON.stringify(tracks)).should.deep.equal(allTracks);
   });
 
-  it('should disable all text tracks', function () {
-    hlsAdapterInstance._videoElement = {
-      textTracks: hls_tracks.subtitles,
-      removeEventListener: () => {}
-    };
-    hlsAdapterInstance._disableAllTextTracks();
-    for (let i = 0; i < hlsAdapterInstance._videoElement.textTracks.length; i++) {
-      hlsAdapterInstance._videoElement.textTracks[i].mode.should.equal('disabled');
-    }
-  });
-
-  it('should hide the active text track', function () {
```

Comment:
why to remove this test?

Reply:
Previously we managed the tracks directly ourselves - so it made sense.
Now we use hlsjs API and it requires loading an actual manifest with text tracks - and we will basically be testing internals of hlsjs - which is out of scope for unit testing

Reply:
we just checkinh that `hideTextTrack` does its work

Reply:
I understand but the test doesn't reflect it - in our unit testing we inject the text tracks ourselves and not via hlsjs so we actually don't test correctly.
And now that we moved to using hlsjs API itself then the test is both irrelevant(as the testing is black boxed behind hls.js API) and not working as we are not using hlsjs to add and manage the text tracks in the test itself.